src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -37,11 +37,9 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily
 {
     internal interface ICFamilyAnalyzer : IAnalyzer
     {
```

Comment:
@duncanp-sonar WDYT? Instead of passing more and more null arguments, I thought about just creating a PreAnalysis option, that would run without rulesConfig and won't call an issue consumer.